cli/src/cli/js-tests-scaffold/index.js

```diff
@@ -1,38 +1,20 @@
 // This test file uses the tape testing framework.
 // To learn more, go here: https://github.com/substack/tape
-const test = require('tape');
-
-const { Config, Conductor } = require("@holochain/holochain-nodejs")
+const { Config, Scenario } = require("@holochain/holochain-nodejs")
+Scenario.setTape(require("tape"))
 
 const dnaPath = "./dist/bundle.json"
+const agentAlice = Config.agent("alice")
+const dna = Config.dna(dnaPath)
+const instanceAlice = Config.instance(agentAlice, dna)
+const scenario = new Scenario([instanceAlice])
 
-// closure to keep config-only stuff out of test scope
-const conductor = (() => {
-    const agentAlice = Config.agent("alice")
-
-    const dna = Config.dna(dnaPath)
-
-    const instanceAlice = Config.instance(agentAlice, dna)
-
-    const conductorConfig = Config.conductor([instanceAlice])
-    return new Conductor(conductorConfig)
-})()
-
-// Initialize the Conductor
-conductor.start()
-
-const alice = conductor.makeCaller('alice', dnaPath)
-
-test('description of example test', (t) => {
+scenario.runTape("description of example test", (t, instances) => {
   // Make a call to a Zome function
-  // indicating the capability and function, and passing it an input
-    const addr = alice.call("my_zome", "create_my_entry", {"entry" : {"content":"sample content"}})
-
-    const result = alice.call("my_zome", "get_my_entry", {"address": addr.Ok})
```

Comment:
@Connoropolous this is likely the pattern all tests will be using since it's part of the scaffolding. Do you prefer this over the destructuring method we use everywhere else? While it shows more clearly what's going on, people might not realize they can write it more succinctly.

Reply:
picking which one is low on my list of concerns. I'm up for switching back to destructuring and moving on 👍